libs/SmartSync/src/com/salesforce/androidsdk/smartsync/manager/CacheManager.java

```diff
@@ -93,7 +97,9 @@
      *
      * @param account User account.
      * @return Instance of this class.
+     * @deprecated Will be removed in Mobile SDK 7.0.
      */
+    @Deprecated
```

Comment:
Won't deprecating the class deprecate all its methods?

Reply:
Not AFAIK. It only marks the class deprecated. The methods have to be individually marked deprecated.